docker/README.md

```diff
@@ -1,54 +1,64 @@
 
-    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 intelanalytics/bigdl:0.2.0-spark-2.1.0
+    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 bigdl/bigdl:0.2.0-spark-2.1.0
 
-    sudo docker run -it --rm --net=host -e NotebookPort=12345 intelanalytics/bigdl:0.2.0-spark-2.1.0
+    sudo docker run -it --rm --net=host -e NotebookPort=12345 bigdl/bigdl:0.2.0-spark-2.1.0
 
 ### If you need http and https proxy in your environment:
 
-    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port intelanalytics/bigdl:default
+    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port bigdl/bigdl:default
 
-    sudo docker run -it --rm --net=host -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port  intelanalytics/bigdl:default
+    sudo docker run -it --rm --net=host -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port  bigdl/bigdl:default
 
-    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port  intelanalytics/bigdl:0.2.0-spark-2.1.0
+    sudo docker run -it --rm -p 12345:12345 -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port  bigdl/bigdl:0.2.0-spark-2.1.0
 
-    sudo docker run -it --rm --net=host -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port intelanalytics/bigdl:0.2.0-spark-2.1.0
+    sudo docker run -it --rm --net=host -e NotebookPort=12345 -e http_proxy=http://your-proxy-host:your-proxy-port -e https_proxy=https://your-proxy-host:your-proxy-port bigdl/bigdl:0.2.0-spark-2.1.0
 
 ### You can also start the container first
 
-    sudo docker run -it --rm --net=host intelanalytics/bigdl:default bash
+    sudo docker run -it --rm --net=host bigdl/bigdl:default bash
 
 ### In the container, after setting proxy and ports, you can start the Notebook by:
 
-    /opt/work/start-notebook
+    /opt/work/start-notebook.sh
+
+## BigDL in Docker
+
+### The work dir is /opt/work
+
+#### download-bigdl.sh is used for downloading BigDL distributions.
+#### start-notebook.sh is used for starting the jupyter notebook. You can specify the environment settings and spark settings to start a specified jupyter notebook.
+#### bigdl-${BIGDL_VERSION} is the BigDL home of BigDL distribution.
+#### dist-spark-x.x.x-scala-x.x.x-linux64-x.x.x-dist.zip is the zip file of BigDL distribution.
+#### spark-${SPARK_VERSION} is the Spark home.
```

Comment:
Whar are the packages/libs installed?
java folder? tutorial folder?

Reply:
ok.